pilot/pkg/model/gateway.go

```diff
@@ -293,13 +304,17 @@ func CheckDuplicates(hosts []string, knownHosts sets.Set) []string {
 // hosts on the two servers start differing -- necessitating the need for two different RDS routes.
 func gatewayRDSRouteName(server *networking.Server, cfg config.Config) string {
 	p := protocol.Parse(server.Port.Protocol)
+	bind := ""
+	if server.Bind != "" {
+		bind = "." + server.Bind
```

Comment:
Is this good enough or a better way to distinguish the route? For unix it will come like `http.80.unix:///var/run/test/socket` and for ip it will come like `http.80.127.0.0.1`

Reply:
This seems like the best option to me